mlrun/model_monitoring/controller.py

```diff
@@ -443,154 +418,71 @@ def model_endpoint_process(
                 )
 
                 for start_infer_time, end_infer_time in batch_window.get_intervals():
-                    # start - TODO : delete in 1.9.0 (V1 app deprecation)
-                    try:
-                        # Get application sample data
-                        offline_response = cls._get_sample_df(
-                            feature_set=m_fs,
+                    prediction_metric = tsdb_connector.read_predictions(
```

Comment:
I'm somewhat worried about this frequent check, especially with TDEngine - see [ML-7806](https://iguazio.atlassian.net/browse/ML-7806).
Is there a way to remove/reduce this read operation?

Reply:
Do you have different suggestion?

Reply:
Let's keep it for now